### DISCUSSION TITLE
Azure DevOps Build issues MacOs agent issue

### DISCUSSION THREAD

**Discussion by bb281** [shinyorg/shiny#678] [Q&A] [answered]
Anyone have issues with building Shiny 2.0 on Azure DevOps MacOS agent? Trying to avoid creating a private build agent. I picked the "latest" option for the MSBuild option. Error: "Projects using Shiny cannot build using MSBuild '16.6.0'. MSBuild '16.8.0' or later is required".

**Comment by aritchie** [comment, marked as answer]
It's listed on my blog what to do in this article: https://allanritchie.com/posts/shiny20

**Reply by bb281** [reply]
Thank you!
